docs/administering/configuring/integration_settings.md

```diff
@@ -104,23 +80,12 @@ The Manifold API handles OAuth callbacks. For Google, the callback route is loca
 
 Manifold Google OAuth requires **Setup Consent** configured with your email, your authorized Manifold domain, and the callback URL entered in the **Authorized redirect URIs** field. The Manifold Integration page has fields to input both **Client ID** and **Client secret**.
 
-Alternatively, if you manage settings in the environment (`MANAGE_SETTINGS_FROM_ENV=1` in your `.env file`), you should set the corresponding settings in `.env`:
-
-```
-# Google OAuth Integration
-MANIFOLD_SETTING_INTEGRATIONS_GOOGLE_OAUTH_CLIENT_ID=
-MANIFOLD_SETTING_SECRETS_GOOGLE_OAUTH_CLIENT_SECRET=
-```
+These settings can also be [managed from environment variables](/docs/administering/configuring/managing_settings#managing-settings-with-environment-variables).
 
 ## Google Analytics
 
 Manifold ships with its own [built-in analytics reporting tool](../../backend/analytics.md), independent of third party systems. However, if you would like to secure usage data from Google as well, you can set up an analytics property in the Google Console. That property will enable Google ti return usage information about your instance in the Google Analytics interface.
```

Comment:
Looks like we have a misspelling here: `ti` instead of `to`.

Reply:
I can fix real quick.

Reply:
Thanks!